hoomd/md/constrain.py

```diff
@@ -4,192 +4,49 @@
 # Maintainer: joaander / All Developers are free to add commands for new
 # features
 
-R""" Constraints.
+r""" Constraints.
 
-Constraint forces constrain a given set of particle to a given surface, to have
-some relative orientation, or impose some other type of constraint.
+Constraint forces can constrain particles to be a set distance from each other,
+to have some relative orientation, or impose other types of constraint.
 
 As with other force commands in hoomd, multiple constrain commands can be issued
 to specify multiple constraints, which are additively applied.
```

Comment:
```suggestion
```
Force objects are not "commands" any longer. In any case, this is explained in the `md.Integrator` docs and doesn't need to be mentioned here.